WBooks/Info.plist

```diff
@@ -60,5 +60,9 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSCameraUsageDescription</key>
+	<string>Se necesita la camara para seleccionar la fotografia del libro</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>Se necesita la camara para seleccionar la fotografia del libro</string>
```

Comment:
PhotoLibrary don't use the camera 🤔

Reply:
I got confused, when copying and pasting I forgot to modify it, I already corrected it by specifying that the gallery was used